helm_deploy/manage-hmpps-auth-accounts/templates/ingress.yaml

```diff
@@ -34,7 +34,7 @@ metadata:
          setvar:tx.paranoia_level=2"
     {{- end }}
     nginx.ingress.kubernetes.io/custom-http-errors: "418"
-    {{ if .Values.ingress.enable_whitelist }}nginx.ingress.kubernetes.io/whitelist-source-range: {{ include "app.joinListWithComma" .Values.whitelist | quote }}{{ end }}
+    {{ if .Values.ingress.enable_allowlist }}nginx.ingress.kubernetes.io/whitelist-source-range: {{ include "app.joinListWithComma" .Values.allowlist | quote }}{{ end }}
```

Comment:
not sure why this line wasn't just deleted if set to `false` in all envs

Reply:
Could have done, decided to leave it as  it's just a templated option.